package.json

```
@@ -29,5 +31,17 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "axe-core": "^4.9.1",
+    "cypress": "^13.9.0",
+    "cypress-axe": "^1.5.0"
+  },
+  "bugs": {
+    "url": "https://github.com/wlsf82/EngageSphere-Test-Design-Masterclass-TAT/issues"
+  },
+  "main": "index.js",
```

Visto que não há nenhum arquivo `index.js`, que tal definir o arquivo de configurações do Cypress como o arquivo principal?

```suggestion
  "main": "cypress.config.js",
```